plonky2/src/fri/mod.rs

```diff
@@ -15,6 +15,7 @@ mod validate_shape;
 pub mod verifier;
 pub mod witness_util;
 
+/// A configuration for the FRI protocol.
```

Comment:
Should we provide a sane `Default` impl for this you think?

Reply:
This one may be less relevant, because typically `StarkConfig` and `CircuitConfig` may deserve their own `Default` impl, but they have drastically different `FriConfig`, one tailored for proving time, the other one for proof size and recursion efficiency. So I'd rather not here.

Reply:
Fair, your call. The counter argument would be: if incompetent people try to use some code, then we should help them out by providing a default that is at least safe-ish so they don't accidentally run with unsafe parameters (I don't know if this can happen here).

Reply:
We can't really have unsafe parameters here, because if for some reasons those were unsound (typically not matching the targeted security level), one could not create a `CircuitBuilder` instance, needed to construct circuits (it checks it).

The only risk is that someone may use a configuration not suited for their application, but well in general I wouldn't recommend people not familiar at all with this kind of things to even think using such library for anything else than fun/learning purposes 🙃